composer.json

```diff
@@ -12,25 +12,25 @@
     ],
     "require": {
         "php": "^7.2",
-        "friendsofsymfony/rest-bundle": "^2.5",
+        "friendsofsymfony/rest-bundle": "^2.8 || ^3.0",
         "jms/serializer-bundle": "^3.0",
-        "php-task/php-task": "^1.3.3",
-        "php-task/task-bundle": "^1.3",
-        "sulu/sulu": "^2.0.6",
-        "symfony/config": "^4.3",
-        "symfony/dependency-injection": "^4.3",
+        "php-task/php-task": "^1.3.2",
+        "php-task/task-bundle": "dev-enhancement/symfony-5",
```

Comment:
@Prokyonn can you have a look at this and check maybe with @wachterjohannes whats missing: https://github.com/php-task/TaskBundle/pull/50